packages/story-editor/src/components/panels/design/videoOptions/useVideoTranscoding.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { useLocalMedia } from '../../../../app';
+import useVideoTrim from '../../../videoTrim/useVideoTrim';
+import useFFmpeg from '../../../../app/media/utils/useFFmpeg';
+
+function useVideoTranscoding({ resource, elementId, isSingleElement = true }) {
```

Comment:
Name of this hook feels a little generic here. Can we do better? @swissspidy any thoughts?

Reply:
I like @swissspidy's suggestion of `useVideoElementTranscoding`. I was struggling a bit with naming here, have no better suggestions.

`useTrimOrMute`? 🤷

Reply:
Let's use `useVideoElementTranscoding` for now then, even though it's a bit long

Reply:
Done in b1f31cba21.